src/styles/antd-overrides/collapse.scss

```diff
@@ -37,3 +37,147 @@
     }
   }
 }
+
+// Top Nav Select
+// Right side of the top nav
+.top-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--background-l0);
+
+  .nav-menu-item {
+    font-weight: 600;
+    color: var(--text-primary) !important;
+  }
+
+  // Resources dropdown and options (dotdotdot) menu
+  .ant-collapse-header {
+    color: var(--text-primary) !important;
+    padding-bottom: 8px !important;
+  }
+
+  .ant-collapse-content {
+    float: right;
+    margin-right: 21px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px 2px;
+    background-color: var(--background-l0);
+    border-radius: 2px;
+  }
+  .ant-collapse-content-box {
+    border: 1px solid var(--stroke-secondary);
+    box-shadow: none !important;
+    padding: 0;
+    width: 100%;
+    background-color: var(--background-l0);
+  }
+  .ant-select-arrow {
+    display: none;
+  }
+  .nav-dropdown-item {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    border: none;
+    padding-left: 10px;
+    width: 100%;
+    height: 40px;
```

Comment:
**issue (non-blocking)**: hopefully we can get to a place where there are less magic numbers regarding heights. Non blocking for this PR though :)

Reply:
Yep gotcha!